lib/const.rb

```diff
@@ -62,6 +62,8 @@
   HOME = CREW_PREFIX + Dir.home
 end
 
+CREW_ARCHIVE_DEST = ENV.fetch('CREW_ARCHIVE_DEST', Dir.pwd)
```

Comment:
Does `Dir.pwd` called from here return the correct current working directory?

Reply:
Yes:
```bash
chronos@mbp113-x86_64 /usr/local/lib/crew/packages $ crew const CREW_ARCHIVE_DEST
CREW_ARCHIVE_DEST=/usr/local/lib/crew/packages
chronos@mbp113-x86_64 /usr/local/lib/crew/packages $ cd ..
chronos@mbp113-x86_64 /usr/local/lib/crew $ crew const CREW_ARCHIVE_DEST
CREW_ARCHIVE_DEST=/usr/local/lib/crew
```

Reply:
and setting it works correctly too:
```
chronos@mbp113-x86_64 /usr/local/lib/crew $ CREW_ARCHIVE_DEST=/usr/local/tmp crew const CREW_ARCHIVE_DEST
CREW_ARCHIVE_DEST=/usr/local/tmp
```

Reply:
Nice!